src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_list_item.scss

```diff
@@ -48,8 +48,9 @@ $thumbnail-width: 105px;
     @include container;
     @include rem(padding, 1.25em $padding);
     display: block;
-    border-bottom: 2px solid $color-structure-normal;
     position: relative;
+    margin-bottom: 1em;
+    box-shadow: 0 0 1em $color-structure-introvert;
```

Comment:
@xi and I discussed the matter. I originally thought we should follow Nico Di Marco's design but it turned out to be unclear.